contents/docs/llm-analytics/installation/anthropic.mdx

```diff
@@ -50,6 +88,10 @@ const client = new Anthropic({
 
 > **Note:** This also works with the `AsyncAnthropic` client as well as `AnthropicBedrock`, `AnthropicVertex`, and the async versions of those.
 
+</Step>
+
+<Step title="Make Anthropic calls" badge="required">
```

Comment:
```suggestion
<Step title="Update Anthropic calls" badge="required">
```

Not sure, maybe this should be to update existing anthropic calls?

Or rather, maybe we should mention that the signature is the same, you just need to replace the client?

Not exactly sure here